token/services/db/sql/common/tokens.go

```diff
@@ -1238,10 +1230,13 @@ func (t *TokenTransaction) StoreToken(ctx context.Context, tr driver.TokenRecord
 
 func (t *TokenTransaction) SetSpendable(ctx context.Context, tokenID token.ID, spendable bool) error {
 	span := trace.SpanFromContext(ctx)
-	query := fmt.Sprintf("UPDATE %s SET spendable = $1 WHERE tx_id = $2 AND idx = $3;", t.table.Tokens)
-	logger.Infof(query, spendable, tokenID.TxId, tokenID.Index)
+	query, args := q.Update(t.table.Tokens).
+		Set("spendable", spendable).
+		Where(cond.And(cond.Eq("tx_id", tokenID.TxId), cond.Eq("idx", tokenID.Index))).
+		Format(t.ci)
+	logger.Infof(query, args...)
```

Comment:
`logger.Info(query, args)`

Make sure it's clear what the difference between `Info` and `Infof` is.